db/migrations/20180912171047_create_cat_file.sql

```diff
@@ -1,21 +1,23 @@
 -- +goose Up
-CREATE TABLE maker.cat_file_chop_lump
+CREATE TABLE maker.cat_file_chop_lump_dunk
```

Comment:
unfortunately, we can't modify an earlier migration if we're looking to have this applied to `beta`. This should work for `staging` though 👍

Reply:
Oh cool. I didn't realize we could alias it in postgraphile. That sounds like a better bet.